contracts/math/SafeMath.sol

```diff
@@ -35,7 +54,18 @@ library SafeMath {
     }
 
     /**
-    * @dev Subtracts two numbers, reverts on overflow (i.e. if subtrahend is greater than minuend).
+    * @dev Integer division of two signed integers truncating the quotient, reverts on division by zero.
+    */
+    function div(int256 a, int256 b) internal pure returns (int256) {
+        // Solidity only automatically asserts when dividing by 0
+        require(b != 0 && b != -1 && a != INT256_MIN);
```

Comment:
Same as before, split `require`s and explain why. 

(this also improves `solidity-coverage`'s report, since each `require` is a single branching point, regardless of its contents).

Reply:
@nventuro Also, great catch! I wrote this wrong... This should be:

```solidity
require(b != 0, "divide by zero");
require(!(b == -1 && a == INT256_MIN), "division overflow");
```